src/external/adobe_coeff.c

```diff
@@ -257,7 +257,7 @@ static void dt_dcraw_adobe_coeff(const char *name, float cam_xyz[1][12])
     { "Nikon D100", { 5902,-933,-782,-8983,16719,2354,-1402,1455,6464 } },
     { "Nikon D1H", { 7577,-2166,-926,-7454,15592,1934,-2377,2808,8606 } },
     { "Nikon D1X", { 7702,-2245,-975,-9114,17242,1875,-2679,3055,8521 } },
-    { "Nikon D1", { 16772,-4726,-2141,-7611,15713,1972,-2846,3494,9521 } }, /* multiplied by 2.218750, 1.0, 1.148438 */
+    { "Nikon D1", { 7559,-2130,-965,-7611,15713,1972,-2478,3042,8290 } },
```

Comment:
@LebedevRI 
Darktable did not open the image without this change. I wanted to test if this change is really useless than I would revert. 
But there is something strange going on. Today it does not open the image anymore, even with my nikon_d1 branch.
In https://github.com/darktable-org/darktable/blob/master/src/iop/temperature.c#L942

```
    // Only check the first three values, the fourth is usually NAN for RGB
    int num_coeffs = (module->dev->image_storage.flags & DT_IMAGE_4BAYER) ? 4 : 3;
    for(int k = 0; k < num_coeffs; k++)
    {
      if(!isnormal(module->dev->image_storage.wb_coeffs[k]) || module->dev->image_storage.wb_coeffs[k] == 0.0f)
      {
        found = 0;
        break;
      }
    }

```
4 coeffs are checked, but the 4th is NaN.

```
coeff[0] = 0.960152
coeff[1] = 1.000000
coeff[2] = 1.041009
coeff[3] = nan

```
and I get

`[temperature] failed to read camera white balance information from `RAW_NIKON_D1.NEF'!`